src/interfaces/history-item-interface.ts

```diff
@@ -43,11 +43,17 @@ export interface HistoryItemIF {
   // alterations only
   courtOrderNumber?: string
   isArrangement?: boolean
+  isCompletedAlteration?: boolean
   isFutureEffectiveAlteration?: boolean
   isFutureEffectiveAlterationPending?: boolean
   toLegalType?: CorpTypeCd
   fromLegalType?: CorpTypeCd
 
+  // dissolutions only
+  isCompletedDissolution?: boolean
+  isFutureEffectiveDissolution?: boolean
+  isFutureEffectiveDissolutionPending?: boolean
```

Comment:
I debated whether to just have `isComplete`, `isFutureEffective` and `isFutureEffectivePending` and then force the template to check the entity type, but that's actually much messier in the template (plus, the template doesn't have to re-compute all the conditions when it re-displays due to reactivity). So, this is actually cleaner IMHO.

Also, note that these states are now consistent between IAs/Alterations/Dissolutions.